### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by loicteixeira**
Description in `CHANGELOG.md`.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,20 @@
 > All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased](https://github.com/springload/wagtaildraftail/compare/v0.3.0...HEAD)
```
Comment (thibaudcolas) [inline CHANGELOG.md:6]: This should be v0.4.0, since we don't do any form of release planning.

File: README.rst [hunk 2]
```diff
@@ -108,8 +108,8 @@ Here is a sample configuration file. This should live in your Django settings.
     DRAFT_ENTITY_TYPE_DOCUMENT = {'label': 'Document', 'type': ENTITY_TYPES.DOCUMENT, 'icon': 'icon-doc-full'}
 
     WAGTAILADMIN_RICH_TEXT_EDITORS = {
-        'default_draftjs': {
-            'WIDGET': 'wagtaildraftail.widgets.JsonTextArea',
+        'default_drafftail': {
```
Comment (thibaudcolas) [inline README.rst]: Typo – `drafftail`.

File: wagtaildraftail/blocks.py [hunk 3]
```diff
@@ -12,7 +12,7 @@ class DraftailTextBlock(RichTextBlock):
     """
     StreamField block to render a rich text editor powered by Draftail.
     """
-    def __init__(self, required=True, help_text=None, editor='default_draftjs', **kwargs):
+    def __init__(self, required=True, help_text=None, editor='default_drafftail', **kwargs):
```
Comment (thibaudcolas) [inline wagtaildraftail/blocks.py]: Typo – `drafftail`.

File: wagtaildraftail/fields.py [hunk 4]
```diff
@@ -10,7 +10,7 @@ class DraftailTextField(models.TextField):
     empty_values = list(EMPTY_SERIALIZED_JSON_VALUES)
 
     def __init__(self, *args, **kwargs):
-        self.editor = kwargs.pop('editor', 'default_draftjs')
+        self.editor = kwargs.pop('editor', 'default_drafftail')
```
Comment (thibaudcolas) [inline wagtaildraftail/fields.py]: Typo - `drafftail`

**Review by thibaudcolas — DISMISSED** [review]
Few typos, looks good to me otherwise!

File: CHANGELOG.md (hunk 1 above)
Comment (thibaudcolas) [reply CHANGELOG.md:6]: Discussed in person – maybe we should have a formal "Unreleased" section always at the top of our changelogs.

File: README.rst (hunk 2 above)
Comment (loicteixeira) [reply README.rst]: Err what a stupid mistake. I'll fix this and merge. Thanks.

**Comment by loicteixeira** [comment]
Build seems to fail on ESLint trying to lint some `node_modules`? It seems unrelated to the PR so I'll merge for now (this work is needed for further development) and we can investigate later.

**Comment by thibaudcolas** [comment]
Looks like the failure is due to nvm not installing the appropriate node version. Might be an infrastructure issue on their end, definitely not related to this PR.

**Pull request merged**